### DISCUSSION TITLE
Can people request existing boomarklets that other people made?

### DISCUSSION THREAD

**Discussion by Samq64** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#188] [Q&A] [answered]
Can people request existing bookmarklets that other people have made (e.g. World_Languages' thumbnail setter)  and then add the original bookmarklet owners to the contributors list?

**Comment by Shluffy** [comment, marked as answer]
You can make any bookmarklet and then add it. For a bookmarklet that you did not make, it may be added, but that will have to get approval from the bookmarklet's owner, and that I would look into before merging a pr.

**Comment by imlens** [comment]
You also can ask us for it in issues and label it as a feature request
